src/Agent.Sdk/Knob/AgentKnobs.cs

```diff
@@ -410,5 +410,12 @@ public class AgentKnobs
             new RuntimeKnobSource("AGENT_USE_NODE"),
             new EnvironmentKnobSource("AGENT_USE_NODE"),
             new BuiltInDefaultKnobSource(string.Empty));
+
+        public static readonly Knob DrainQueuesAfterTask = new Knob(
+            nameof(DrainQueuesAfterTask),
+            "Forces the agent to drain queues after each task",
+            new RuntimeKnobSource("AGENT_DRAIN_QUEUES_AFTER_TASK"),
+            new EnvironmentKnobSource("AGENT_DRAIN_QUEUES_AFTER_TASK"),
+            new BuiltInDefaultKnobSource("true"));
```

Comment:
>true [](http://example.com/codeflow?start=42&length=4)

For ADO feature flags, we can't set to "true" by default.  (technical, we can but the mechanism will cause the FF to be reset at next deployment- - not what we want).  

So we'll need to use an "inverse feature flag" in this case.  e.g. it should be "DisableDrainQueuesAfterTask", so the "false" behavior is default

Reply:
I have reversed this knob, and renamed it to `AGENT_DISABLE_DRAIN_QUEUES_AFTER_TASK`